metrics/engine/engine.go

```diff
@@ -245,7 +246,9 @@ func (me *MetricsEngine) evaluateThresholds(ignoreEmptySinks bool) (breachedTher
 			shouldAbort = true
 		}
 	}
-
+	if len(breachedThersholds) > 0 {
+		sort.Strings(breachedThersholds)
+	}
 	me.logger.Debugf("Thresholds on %d metrics breached: %v", len(breachedThersholds), breachedThersholds)
```

Comment:
Should we move the other operations below in the if block?

Reply:
hmm maybe the logging, with a separate log if no thresholds were breached :thinking: but the `atomic.StoreUint32()` and return should stay outside of the `if`, right?

Reply:
Yeah, this comment is for logline.